src/main/resources/linking/en.yml

```diff
@@ -1,3 +1,25 @@
+# Minecraft to Discord account linking
+# These are the options pertaining to how linking a Minecraft account to a Discord account functions
+#
+# MinecraftDiscordAccountLinkedConsoleCommands: commands to run when an account is linked, see below for possible placeholders
+# MinecraftDiscordAccountUnlinkedConsoleCommands: commands to run when an account is unlinked, see below for possible placeholders
+# %minecraftplayername%: player's Minecraft username
+#                         example: Notch
+# %minecraftuuid%:       player's uuid
+#                         example: you know what a uuid looks like
+# %discordid%:           linked discord account's id
+#                         example: 12345678901234567890
+# %discordname%:         linked discord account's username
+#                         example: Notch
+#
+# MinecraftDiscordAccountLinkedRoleNameToAddUserTo: the name of a discord role to add a discord user to when they link their account
+# MinecraftDiscordAccountLinkedAllowRelinkBySendingANewCode: allows sending a new code to the bot to unlink and relink with the new code
+#
+MinecraftDiscordAccountLinkedConsoleCommands: ["", "", ""]
+MinecraftDiscordAccountUnlinkedConsoleCommands: ["", "", ""]
+MinecraftDiscordAccountLinkedRoleNameToAddUserTo: "Linked"
+MinecraftDiscordAccountLinkedAllowRelinkBySendingANewCode: false
+
```

Comment:
Should we be moving `MinecraftDiscordAccount...` options into `linking.yml`? I don't remember who recommended (probably me, since I committed it) but I'm not sure it's a good idea anymore (unless at some point option migration between config files was implemented?) Would probably be best to just move them back into `config.yml`

Reply:
Config migration is copies lines from the same file as of now. For existing users moving it might confuse them if they can't find the option anymore, for new users it might help with finding those options. 

It's probably not a good idea to reset those options for people, so I'd suggest moving those back to the config.

Reply:
https://github.com/DiscordSRV/DiscordSRV/commit/544f53ad4814a2e1ab6c9cb563a7f7af5165a891